### ISSUE TITLE
Комментарий ревьюера

### ISSUE THREAD

**Issue by Sekai-no-uragawa** [frauRat/Project_one#1]
## Комментарий ревьюера

Привет, Елена! Меня зовут Михаил Костин, я буду проверять этот проект. Спасибо за проделанную работу! Предлагаю общаться на «ты».

*Давай смотреть, что получилось!*

В твоем проекте выполнено большинство необходимых пунктов:
- ✔️Работа над проектом велась последовательно и разбита на отдельные коммиты
- ✔️Описание коммитов соответствуют содержимому
- ✔️Заполнен файл описания проeкта readme.md

❌Однако остался один момент, необходимый для принятия проекта — **"Создать issue с возможными улучшениями проекта."**

Дополнительно, но необязательно (если хочешь улучшить этот проект или чтобы учесть при самостоятельном использовании github):
- В readme проекта, кроме описания постановки задачи, здорово описать и полученные результаты — какие метрики получились, какие модели использовались и т.д.
- readme .md — markdown файл, у него простой синтаксис разметки, оформлять можно очень красиво и функционально, [Шпаргалка](https://texterra.ru/blog/ischerpyvayushchaya-shpargalka-po-sintaksisu-razmetki-markdown-na-zametku-avtoram-veb-razrabotchikam.html)
- Заполняй About репозитория, так будет легче производить навигацию по твоему github.

## Итоговый комментарий ревьюера
Жду твоих правок, общаться предлагаю в этом issue
Надеюсь, комментарии помогут закончить проект. Постараюсь повторно проверить твой проект как можно быстрее.

**Comment by frauRat** [comment]
Михаил, добрый день!
У меня тут техническая ошибка возникла - почему то не могу ссылку заново отправить в Практикуме, issue то я создала

**Comment by Sekai-no-uragawa** [comment]
Да, вижу, но к сожалению не могу принять без открытия тетрадки из практикума, сейчас стало выдавать ошибку при ее открытии.

**Comment by frauRat** [comment]
Вчера тех поддержка пыталась целый день решить вопрос, но пока безрезультатно.. надеюсь сегодня получится

**Comment by Sekai-no-uragawa** [comment]
Привет, к сожалению, такие ошибки на платфоре иногда случаются :( Нашел твой тикет в поддержку, вроде получилось решить проблему, задание принято. Также передаю слова поддержки: "Напомните, пожалуйста, студентке на всякий случай, чтобы не забыла пройти заключение и обратную связь к спринту, чтобы  открылся следующий."
